src/test/regress/expected/multi_citus_tools.out

```diff
@@ -629,6 +629,19 @@ SELECT citus_check_connection_to_node('localhost', :worker_2_port);
 (1 row)
 
 \c - - - :master_port
+SELECT * FROM citus_check_cluster_node_health() ORDER BY 1,2,3,4;
+WARNING:  connection to the remote node localhost:xxxxx failed with the following error:
+WARNING:  connection to the remote node localhost:xxxxx failed with the following error:
+WARNING:  connection to the remote node localhost:xxxxx failed with the following error:
+WARNING:  connection to the remote node localhost:xxxxx failed with the following error:
+ from_nodename | from_nodeport | to_nodename | to_nodeport | success
+---------------------------------------------------------------------
+ localhost     |         57637 | localhost   |       57637 | f
+ localhost     |         57637 | localhost   |       57638 | f
+ localhost     |         57638 | localhost   |       57637 | f
+ localhost     |         57638 | localhost   |       57638 | f
+(4 rows)
+
```

Comment:
This test has a broken result when run on the test suite.

On a local cluster, I see the expected results:
```
hanefi=# SELECT * FROM citus_check_cluster_node_health();
LOG:  statement: SELECT * FROM citus_check_cluster_node_health();
NOTICE:  issuing SELECT citus_check_connection_to_node('localhost', 9701)
DETAIL:  on server hanefi@localhost:9701 connectionId: 1
DEBUG:  issuing SELECT 1
DETAIL:  on server hanefi@localhost:9701 connectionId: 9
DETAIL:  from localhost:9701
NOTICE:  issuing SELECT citus_check_connection_to_node('localhost', 9702)
DETAIL:  on server hanefi@localhost:9701 connectionId: 1
DEBUG:  issuing SELECT 1
DETAIL:  on server hanefi@localhost:9702 connectionId: 10
DETAIL:  from localhost:9701
NOTICE:  issuing SELECT citus_check_connection_to_node('localhost', 9701)
DETAIL:  on server hanefi@localhost:9702 connectionId: 2
DEBUG:  issuing SELECT 1
DETAIL:  on server hanefi@localhost:9701 connectionId: 9
DETAIL:  from localhost:9702
NOTICE:  issuing SELECT citus_check_connection_to_node('localhost', 9702)
DETAIL:  on server hanefi@localhost:9702 connectionId: 2
DEBUG:  issuing SELECT 1
DETAIL:  on server hanefi@localhost:9702 connectionId: 10
DETAIL:  from localhost:9702
 from_nodename | from_nodeport | to_nodename | to_nodeport | success
---------------+---------------+-------------+-------------+---------
 localhost     |          9701 | localhost   |        9701 | t
 localhost     |          9701 | localhost   |        9702 | t
 localhost     |          9702 | localhost   |        9701 | t
 localhost     |          9702 | localhost   |        9702 | t
```

Reply:
It turned out that I was missing some calls to `FinishConnectionEstablishment`. Fixed that in 658cb77